packages/commonwealth/client/scripts/helpers/formValidationMessages.ts

```diff
@@ -1,5 +1,5 @@
 export const VALIDATION_MESSAGES = {
-  NO_INPUT: 'No input',
+  NO_INPUT: 'Field required',
```

Comment:
If this is changed by the product team, then I think we should also update it in https://github.com/hicommonwealth/commonwealth/blob/master/packages/commonwealth/client/scripts/views/pages/Community/Groups/common/GroupForm/validations.ts#L4 or maybe move it up (abstraction)

Reply:
actually not, changed it on my own, was not aware of other inputs using the same message. I am reverting it then.